src/System.Runtime.Numerics/tests/BigInteger/add.cs

```diff
@@ -1,160 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using Xunit;
-
-namespace System.Numerics.Tests
-{
-    public class addTest
-    {
-        private static int s_samples = 10;
-        private static Random s_random = new Random(100);
-
-        [Fact]
-        public static void RunAddTests()
-        {
-            byte[] byteArray1 = new byte[0];
-            byte[] byteArray2 = new byte[0];
-
-            // Add Method - Two Large BigIntegers
-            for (int i = 0; i < s_samples; i++)
-            {
-                byteArray1 = GetRandomByteArray(s_random);
-                byteArray2 = GetRandomByteArray(s_random);
```

Comment:
Thanks - I didn't realise that the big integers are *that* big. I've added back the following tests (based on your suggestion):
- 10x big + big
- 10x big + small